Taskfile.yml

```diff
@@ -346,22 +312,159 @@ tasks:
 
       # Port-forward dependency services
       - |
-        {{ .KUBECTL_BIN }} port-forward service/dir-apiserver  8888:8888   -n {{ .HELM_NAMESPACE }} &
+        {{ .KUBECTL_BIN }} port-forward service/dir-apiserver 8888:8888 -n {{ .HELM_NAMESPACE }} &
```

Comment:
Maybe it would be better to create a task for port-forwards, and use it before tests along with defer teardown task of the port-forward.